app/main.py

```diff
@@ -2,24 +2,62 @@
 
 
 class IntegerRange:
-    pass
+    """Data validation descriptor"""
+
+    def __init__(self, min_value: int, max_value: int) -> None:
+        self.min_value = min_value
+        self.max_value = max_value
+
+    def __set_name__(self, instance: object, name: str) -> None:
+        self.private_name = f"_{name}"
```

Comment:
In the __set_name__ method, the instance parameter is not used. If it's not necessary, consider removing it.